python_modules/libraries/dagster-dbt/dagster_dbt/include/setup.py.jinja

```diff
@@ -4,6 +4,13 @@ setup(
     name="{{ project_name }}",
     version="0.0.1",
     packages=find_packages(),
+    {% if use_dbt_project_package_data_dir -%}
+    package_data={
+        "{{ project_name }}": [
+            "dbt-project/*",
```

Comment:
fyi not sure if you expected nested directories here but this might only do one level by default.

Reply:
I did expect nested directories here, but looks like `/*` worked as I expected, since it built in cloud. But I'll use a recursive glob here, since that seems to be supported as well, and makes this behavior explicit: https://setuptools.pypa.io/en/latest/history.html#v62-3-0